packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -101,12 +114,14 @@ const CartSecondaryButtonGroup = ({
                             />
                         </Button>
                     )}
-                    <Button variant="link" size="sm" onClick={() => onEditClick(variant)}>
-                        <FormattedMessage
-                            defaultMessage="Edit"
-                            id="cart_secondary_button_group.action.edit"
-                        />
-                    </Button>
+                    {shouldShowEditButton() && (
```

Comment:
How about this? We don't need to have our own logic to detect whether `variant` is actually a variation or bundled product.. because the API already does it for us.

```suggestion
                    {!variant.type.item && (
```

In this case, `type.item` refers to the standard product type. See doc for more details: https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-products?meta=getProduct

Reply:
Instead of an allowlist (if it's of type A, B, C, then show the Edit button), I think we should do the inverse: if it's _not_ standard product, then show the Edit button.

There are other product types besides variation and bundled products that need the Edit button. For example, product set is one of them.

From the docs:
> `type`
> The product type information. Can be one or more of the following values: item, master, variation_group, variant, bundle, and set.

Reply:
Yes, this sounds like a better approach. Thanks Vincent!